main.cpp

```diff
@@ -59,85 +59,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:         dog
+//  action 1:     bark
+dog.bark();
+//  action 2:     run
+dog.run();
+//  action 3:     sleep
+dog.sleep(); 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:         ball
+//  action 1:     bounce
+ball.bounce();
+//  action 2:     stop
+ball.stop();
+//  action 3:     roll
+ball.roll();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:         cellPhone
+//  action 1:     ring
+cellPhone.ring();
+//  action 2:     vibrate
+cellPhone.vibrate();
+//  action 3:     discharge
+cellPhone.discharge();
  
+//  4)            
+//  Noun:         compiler 
+//  action 1:     compile
+compiler.compile();
+//  action 2:     stop
+compiler.stop();
```

Comment:
as I said earlier regarding `ball.stop()`, the same thing applies here.  it's not really an action.  add what is being stopped or choose a different action.
- [ ] revise